test/src/test/resources/fixtures/hydration/polymorphic hydrations/solitary-polymorphic-hydration-when-hook-returns-null.yml

```diff
@@ -134,7 +134,6 @@ serviceCalls:
         query: |
           query {
             humanById(id: "HUMAN-0") {
-              __typename__type_filter____typename: __typename
```

Comment:
Just note here, this is deleted because the ENF now merges the `__typename` field correctly. So the field is no longer completely removed and thus triggering the typename filter transform.

Can explain more in call, too lazy to type out full details.